### DISCUSSION TITLE
RPC is supported in latest DDS release?

### DISCUSSION THREAD

**Discussion by mahesh2392** [eProsima/Fast-DDS#2909] [Q&A]
Hello guys, I am exploring eProsima DDS and I am wondering that current DDS release supports RPC or not ? As I seen it in previous releases but haven't seen any examples in latest releases. Thanks.

**Comment by EduPonz** [comment]
I'm not sure I understand your question, let me try

* It's possible to have an RPC mechanism on top of DDS. This is typically implemented using a couple of topics.
* [eProsima/RPC](https://github.com/eProsima/RPC) provides a partial implementation of the [DDS-RPC](https://www.omg.org/spec/DDS-RPC/1.0/About-DDS-RPC/) specification. This project does not receive much support, but it is still functional and should work for you.
